### ISSUE TITLE
White line in nautilus - Ubuntu 16.04

### ISSUE THREAD

**Issue by comevolo** [snwh/paper-gtk-theme#319]
I've found a problem with your theme in ubuntu 16.04. There's a white line in nautilus under the window bar
![schermata del 2016-04-23 17-28-40](https://cloud.githubusercontent.com/assets/18631952/14762385/157ccb6a-0979-11e6-8a42-a3bc8b94e0d0.png)

**Comment by snwh** [comment]
Yeah, the rounded corners and the highlight on header bars is deliberate, the real issue here is that Unity's window manager doesn't support this. My recommendation is to add the (completely safe) GNOME3 PPA:

    sudo add-apt-repository ppa:gnome3-team/gnome3

**Comment by comevolo** [comment]
i'll try asap. thank you!

**Comment by cmoud94** [comment]
@snwh Not completely safe, this ppa caused white rectangle around nautilus and my desktop icons were really big. At least it doesn't broke my system, so it's safe from that point of view.

**Comment by snwh** [comment]
@cmoud94 I could go with: "Don't use Unity" in that case. ;) Compiz doesn't support header bars as well as other window managers, which is the main issue here.

**Comment by jyotishp** [comment]
Updating from GNOME3 PPA didn't fix it.

**Comment by poltertec** [comment]
I fixed this by setting box-shadow & radius for titlebar & removing them from header-bar.

```
.titlebar {
   border-radius: 4px 4px 0 0;
   box-shadow: inset 0 1px $top_highlight;
 }
```

**Comment by matteosecli** [comment]
Hello @poltertec , can you please give me more details on how to modify the theme with your fix? As the file I have to edit, the lines, etc (or just a fork of this repo with the fix). I don't know anything at all about css, and my trials had no success. Thanks in advance! :)

**Comment by matteosecli** [comment]
Nevermind, I figured it out. It works for me on Ubuntu 16.04.1.

**Comment by yurividal** [comment]
I have this issue, on Ubuntu 16.10, eventhough i am running latest version of paper. what am i doing wrong?

**Comment by simontong** [comment]
I'm also getting this issue still. I'm using Ubuntu 16.10

![image](https://cloud.githubusercontent.com/assets/1494859/26274188/ca9d5ea0-3d3b-11e7-930f-809422f9bc16.png)
